apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/TraceResource.java

```diff
@@ -100,6 +101,16 @@ public Response getById(@PathParam("id") UUID id) {
                 .block();
     }
 
+    @GET
+    @Path("/workspaces-traces")
```

Comment:
For a more RESTFUL naming, I suggest something such as `/workspace-counts` or similar.

Reply:
updated